packages/core/src/processing/generator/generated_data.rs

```diff
@@ -40,6 +40,7 @@ impl GeneratedData {
     /// * `synthetic_data` - Synthesized data headers (index 0) and records indexes 1...
     /// * `expansion_ratio` - `Synthetic data length / Sensitive data length` (header not included)
     /// * `multi_value_column_metadata_map` - Maps a normalized multi-value header name (such as A_a1)
+    /// * `long_form` - Pivots column headers and value pairs to key-value row entries.
```

Comment:
Maybe this does not need to be here, since it is not a parameter of the `new`.

Reply:
Oops. Not sure why I put that there.